addons/im_livechat/static/src/legacy/widgets/livechat_button.js

```diff
@@ -137,22 +136,6 @@ const LivechatButton = Widget.extend({
         ev.stopPropagation();
         utils.set_cookie('im_livechat_session', utils.unaccent(JSON.stringify(this.messaging.publicLivechatGlobal.publicLivechat.widget.toData()), true), 60 * 60);
     },
-    /**
-     * @private
-     * @param {OdooEvent} ev
-     */
-    _onUpdatedTypingPartners(ev) {
-        ev.stopPropagation();
-        this.messaging.publicLivechatGlobal.chatWindow.widget.renderHeader();
-    },
```

Comment:
you haven't replaced `updated_typing_partners` in this commit, I'm not sure if it's okay to remove it here

Reply:
oops, reverted in this PR